mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionSourceChunkDaoImpl.java

```diff
@@ -137,18 +138,18 @@ public Future<JobExecutionSourceChunk> update(JobExecutionSourceChunk jobExecuti
       pgClientFactory.createInstance(tenantId)
         .update(TABLE_NAME, jobExecutionChunk, new Criterion(idCrit), true, updateResult -> {
           if (updateResult.failed()) {
-            LOGGER.error("Could not update jobExecutionSourceChunk with id {}", jobExecutionChunk.getId(), updateResult.cause());
+            LOGGER.warn("update:: Could not update jobExecutionSourceChunk with id {}", jobExecutionChunk.getId(), updateResult.cause());
             promise.fail(updateResult.cause());
           } else if (updateResult.result().rowCount() != 1) {
             String errorMessage = String.format("JobExecutionSourceChunk with id '%s' was not found", jobExecutionChunk.getId());
```

Comment:
could you add the method name to the error message?

Reply:
added